tyk-pro/README.md

```diff
@@ -2,32 +2,43 @@
 Provides a full Tyk Installation API Management Dashboard and API Gateways with Analytics)
 
 ### Installation
-To install, *first modify the `values.yaml` file and add your license*. Then run the following command from the root of the repository:
-
-	helm install tyk-pro ./tyk-pro -n tyk --wait
-
+```
+helm repo add tyk-helm https://helm.tyk.io/public/helm/charts/
+helm repo update
+```
+Before we proceed with installation of the chart we need to set some custom values. To see what options are configurable on a chart and save that options to a custom values.yaml file run:
+```
+helm show values tyk-helm/tyk-pro > values.yaml
```

Comment:
@vesko-tyk Can we update the instructions of the READMEs 
```
helm show readme tyk-helm/tyk-pro
helm show readme tyk-helm/tyk-hybrid
helm show readme tyk-helm/tyk-headless
```

Reply:
This PR is still not merged, once approved and merged I'll update the actual package instructions as well.